shared/constants/wallets.js

```diff
@@ -326,6 +338,32 @@ const paymentToYourRoleAndCounterparty = (
   }
 }
 
+// Update payment, take all new fields except any that contain the default value
+const emptyPayment = makePayment()
+// $FlowIssue thinks we don't have toJS()
+const keys = Object.keys(emptyPayment.toJS())
```

Comment:
why not just get the keys? `.keySeq().toArray()`

Reply:
I don't think that works with `Record`s

Reply:
`.toSeq().keySeq().toArray()`